.github/workflows/issue_stale.yml

```diff
@@ -10,6 +10,19 @@ jobs:
     runs-on: ubuntu-latest
     if: github.repository_owner == 'vercel'
     steps:
+      - uses: actions/stale@v9
+        id: issue-stale
+        name: 'Mark stale issues, close stale issues'
+        with:
+          repo-token: ${{ secrets.STALE_TOKEN }}
+          ascending: true
+          days-before-issue-stale: 730 # issues with no activity in over two years
+          days-before-issue-close: 7
+          remove-issue-stale-when-updated: true
+          stale-issue-label: 'stale'
+          stale-issue-message: 'This issue has been automatically marked as stale because it has not had recent activity in the last two years. It will be closed in 7 days if no further activity occurs. If you believe this was done in error, please leave a comment. Thank you.'
```

Comment:
```suggestion
          stale-issue-message: 'This issue has been automatically marked as stale due to two years of inactivity. It will be closed in 7 days unless there’s further input. If you believe this issue is still relevant, please leave a comment or provide updated details. Thank you.'
```